css/partner/add-listing.css

```diff
@@ -0,0 +1,135 @@
+.container-upload {
+  width: 805px;
+  height: 255px;
+  margin-left: 217px;
+  margin-top: 31px;
+
+  background: #C4C4C4;
+  border-radius: 10px;
+}
+
+.upload-caption {
+  width: 206px;
+  height: 20px;
+  margin-left: 217px;
+  margin-top: 8px;
+
+  font-family: Poppins-Regular;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 21px;
+
+  color: #2CAD69;
+}
+
+.form-row {
+  width: 609px;
+  margin-left: 243px;
+  margin-top: 28px;
+}
+
+.star-button {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  background: #21374A;
+  width: 39px;
+  height: 39px;
+}
+
+.star-button > i {
+  color: var(--tph-text-secondary);
+  font-size: 20px;
+}
+
+.long-col {
+  margin-bottom: 24px;
+  margin-left: 11px;
+}
+
+.col-no-var {
+  margin-bottom: 11px;
+  margin-left: 33px;
+}
+
+.col-var-inputs {
```

Comment:
Rename.

Also, why do you even have this? One would assume that it would encompass ALL the inputs. But that is not the case in the code. You're wrapping it randomly. I honestly do not understand.

Something like .add-listing-form-inputs would make sense.